bootstrap/kubeadm/docs/external-etcd.md

```diff
@@ -1,49 +1,76 @@
+    * As an example, cross availability zone traffic can cost money on cloud providers. You don't have to deploy etcd across availability zones, but if you do please be aware of the costs.
 
 ## Getting started
 
-To use this, you will need to create an etcd cluster and generate an apiserver-etcd-client key/pair.
-This behaviour can be tested using [`kubeadm`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) and [`etcdadm`](https://github.com/kubernetes-sigs/etcdadm).
+To use this, you will need to create an etcd cluster and generate an apiserver-etcd-client certificate and private key. This behaviour can be tested using [`kubeadm`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) and [`etcdadm`](https://github.com/kubernetes-sigs/etcdadm).
  
 ### Setting up etcd with kubeadm 
 
-CA certificates are required to setup etcd cluster.
-If you already have a CA then the CA's `key` and `crt` must be copied to `/etc/kubernetes/pki/etcd/ca.crt` and `/etc/kubernetes/pki/etcd/ca.key`. 
+CA certificates are required to setup etcd cluster. If you already have a CA then the CA's `crt` and `key` must be copied to `/etc/kubernetes/pki/etcd/ca.crt` and `/etc/kubernetes/pki/etcd/ca.key`. 
 
-If you do not already have a CA then run command `kubeadm init phase certs etcd-ca`. This creates two files
+If you do not already have a CA then run command `kubeadm init phase certs etcd-ca`. This creates two files:
 
- * `/etc/kubernetes/pki/etcd/ca.crt`
- * `/etc/kubernetes/pki/etcd/ca.key`  
+* `/etc/kubernetes/pki/etcd/ca.crt`
+* `/etc/kubernetes/pki/etcd/ca.key`  
 
-These key/pair are used to sign etcd server, peer certificates and eventually apiserver-etcd client. More information on how to setup external etcd with kubeadm can be found [`here`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/#setting-up-the-cluster).
+This certificate and private key are used to sign etcd server, peer certificates and eventually apiserver-etcd-client certificate. More information on how to setup external etcd with kubeadm can be found [here](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/#setting-up-the-cluster).
 
-You would require files `/etc/kubernetes/pki/apiserver-etcd-client.key`, `/etc/kubernetes/pki/apiserver-etcd-client.crt` and `/etc/kubernetes/pki/etcd/server.crt` to setup etcd cluster. These are put in 2 secrets.
+Once the etcd cluster is setup, you will need the following files from the etcd cluster:
 
-```
-# Kubernetes APIServer etcd client certificate
+1. `/etc/kubernetes/pki/apiserver-etcd-client.crt` and `/etc/kubernetes/pki/apiserver-etcd-client.key`
+2. `/etc/kubernetes/pki/etcd/ca.crt`
+
+You'll use these files to create the necessary Secrets on the management cluster (see the "Creating the required Secrets" section).
+
+### Setting up etcd with etcdadm (Alpha)
```

Comment:
/cc @dlipovetsky 
i'm having flashbacks about a similar PR some time ago.
did we end up not wanting to have etcdadm docs yet; i cannot recall?

Reply:
never mind, i see this was present already bellow.
@dlipovetsky PTAL if you can at the update.